sympy/functions/special/tests/test_delta_functions.py

```diff
@@ -39,11 +39,11 @@ def test_DiracDelta():
     assert DiracDelta(sqrt(x)).is_simple(x) is False
     assert DiracDelta(x).is_simple(y) is False
 
-    assert DiracDelta(x*y).simplify(x) == DiracDelta(x)/abs(y)
-    assert DiracDelta(x*y).simplify(y) == DiracDelta(y)/abs(x)
-    assert DiracDelta(x**2*y).simplify(x) == DiracDelta(x**2*y)
-    assert DiracDelta(y).simplify(x) == DiracDelta(y)
-    assert DiracDelta((x - 1)*(x - 2)*(x - 3)).simplify(x) == \
```

Comment:
The above should be left in place until this method is removed in future versions. Otherwise you risk breaking the deprecated code if it isn't tested.